django/contrib/staticfiles/storage.py

```diff
@@ -80,7 +80,7 @@ class HashedFilesMixin:
         (
             "*.css",
             (
-                r"""(?P<matched>url\(['"]{0,1}\s*(?P<url>.*?)["']{0,1}\))""",
+                r"""(?P<matched>url\((?P<quote>['"]{0,1})\s*(?P<url>.*?)(?P=quote)\))""",  # noqa: E501
```

Comment:
After seeking guidance from other contributors about using `# noqa` comments, it seems that we should break the pattern into two lines and remove the comment. How about this?

```python
                r"""(?P<matched>url\((?P<quote>['"]{0,1})"""
                r"""\s*(?P<url>.*?)(?P=quote)\))""",
```

Reply:
This is what I had originally and why I shortened the regex capture group name to just `q`, but this works as well.

Reply:
Yeah, I didn't expect the linting tools to complain. I think the `quote` naming helps though. Thanks again for your patience here!